include/fmt/core.h

```diff
@@ -1384,6 +1385,12 @@ typename buffer_context<Char>::type::iterator vformat_to(
   \rst
   Returns a named argument to be used in a formatting function.
 
+  Usage is analogous to `std::reference_wrapper
+  <https://en.cppreference.com/w/cpp/utility/functional/reference_wrapper>`_,
+  except that rvalues references are not disabled.
```

Comment:
should be `rvalue references`

Reply:
Actually I think it should just be rvalues, since rvalue references are lvalues.

Reply:
Is the rvalue part useful here? I'd drop it altogether.

Reply:
I'll change it to just dangling references. Not all users would know what an rvalue is (or reference_wrapper for that matter)